README.md

```diff
@@ -54,7 +59,8 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+town[:castle][:guests] << town[:residents][1]
+p town[:castle][:guests]
 ```
 
```

Comment:
So your first line of code will put "Belle" in the `guests` array, but how would you delete her from `residents`?
Possible solution

```rb
town[:residents].delete("Belle")
```